nwb_conversion_tools/datainterfaces/spikeglxdatainterface.py

```diff
@@ -56,4 +56,23 @@ def get_metadata(self):
                 )
             )
         )
+
         return ecephys_metadata
+
+
+class SpikeGLXLFPInterface(BaseLFPExtractorInterface):
+    """Primary data interface class for converting the high-pass (ap) SpikeGLX format."""
+
+    RX = SpikeGLXRecordingExtractor
+
+    def get_metadata(self):
+        metadata = dict(
+            Ecephys=dict(
+                LFPElectricalSeries=dict(
+                    name="LFP",
+                    description="Local field potential signal."
+                )
+            )
+        )
```

Comment:
This seems generic. Can it be put [here](https://github.com/catalystneuro/nwb-conversion-tools/blob/452d456d15f0d2107077f28ad73c7b62512af454/nwb_conversion_tools/baselfpextractorinterface.py#L15) instead?

Reply:
Yeah, that's the better place for it. Updated!